generator/03-syndesis-rest.yml.mustache

```diff
@@ -141,13 +141,13 @@
               port: 8181
               scheme: HTTP
             initialDelaySeconds: 180
-            timeoutSeconds: 3
+            timeoutSeconds: 5
           readinessProbe:
             httpGet:
               path: "/health"
               port: 8181
             initialDelaySeconds: 5
```

Comment:
I set everything to 5 seconds, assuming that the mysterious delay is the same for liveness and readiness checks as they both come from the kubelet.